api/candidatesApiClient.ts

```diff
@@ -0,0 +1,10 @@
+﻿export async function getCandidates(): Promise<string[]> {
+    
+    //get json candidates from site
+    
+    //get names from candidates
+    
+    //put into list
+    
+    return ["Michael Walker", "Stephen Jackson"];
+}
```

Comment:
So it looks like there is lots of stuff in this PR that isn't stuff you've been working on and we wouldn't want to commit.

(ie this looks like an early draft of the work that Jamie did)
We'll need to either
- Clear up this PR so that it only includes work that you'd like to merge into master
- Wait for Jamie's PR to happen, and then handle all the merge conflicts.

Which would you prefer?

Reply:
We branched off of Jamie's branch as we thought it would be easier - rather than having to duplicate styling etc.

If I clear this PR to only include the work I've done will that affect the previous branches?

Reply:
nothing you do here will affect Jamie's PR - they are totally separate.

The stuff Jamie did is now quite different to what you've got here, (because he's done a fair bit more work on it)

He has finished and merged that bit of work though now - so it should be easier to merge it back in.

Reply:
We definitely don't want to add all this work again though right - master already has Jamie's work. We don't want to add it again - just add in our new work.